### PR TITLE
Add content to auditlog

### PR DESCRIPTION
**Pull request by 2e2a**

### CODE REVIEW

File: src/adhocracy_core/adhocracy_core/auditing/__init__.py [hunk 1]
```diff
@@ -92,8 +99,17 @@ def audit_resources_changes_callback(request: Request,
...
+        if _is_audit_change(change):
+            action_name = _get_entry_name(change),
+            sheets = _get_content_sheets(change, registry)
+            sheet_data = _get_sheet_data(sheets, request)
+            log_auditevent(change.resource,
+                           action_name,
+                           user_name=user_name,
+                           user_path=user_path,
+                           sheet_data=sheet_data)
+            transaction.commit()
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/auditing/__init__.py]: Not part of this pull request, but why do we need a commit for every single auditlog?

File: src/adhocracy_core/adhocracy_core/sheets/__init__.py [hunk 2]
```diff
@@ -228,7 +228,7 @@ def _store_references(self, appstruct, registry, send_event=True):
                                               registry,
                                               send_event=send_event)
 
-    def serialize(self, params: dict=None):
+    def serialize(self, params: dict=None, add_back_references: bool=True):
         """Get sheet appstruct data and serialize.
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/sheets/__init__.py]: test is missing

File: src/adhocracy_core/adhocracy_core/auditing/__init__.py (hunk 1 above)
Comment (2e2a) [inline src/adhocracy_core/adhocracy_core/auditing/__init__.py]: | Not part of this pull request, but why do we need a commit for every single auditlog?

It was already there, I don't see why. I think we can remove it.
